<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>handle_notifications</class>
 <widget class="QDialog" name="handle_notifications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_views">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>No. of Views:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_accept">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deny">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Deny</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_notifications">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Someone has requested image 1 for kaza view</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
